build.sh

```diff
@@ -123,5 +123,5 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 if [ "$alternateTarget" == "true" ]; then
   "$CLIPATH/dotnet" $SDKPATH/MSBuild.dll "$scriptroot/build.proj" /bl:source-build-test.binlog /flp:v=diag /clp:v=m "$@"
 else
-  "$scriptroot/eng/common/build.sh" --restore --build -c Release --warnaserror false -bl /flp:v=diag "$@"
+  "$scriptroot/eng/common/build.sh" --restore --build -c Release --warnaserror false /bl:Build_$(date +"%m%d%H%M%S").binlog /flp:v=diag "$@"
```

Comment:
I think we should move this to the standard `artifacts/log/Debug` location. Having one binlog in the repo root is not great (in theory source dirs should not be modified) but having them build up makes it worse--harder to navigate the repo since it's harder to see the "real" files.

Good idea to preserve them though!

Reply:
Agreed.